package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "authomatic-redis",
-  "version": "0.0.3",
+  "version": "1.0.0",
```

Comment:
I would bump the version in a separate commit, which you can then also `git tag`.